meganav.yaml

```diff
@@ -863,20 +863,6 @@ get_ubuntu:
               url: /download/server/s390x
             - title: RISC-V
               url: /download/risc-v
-    - title: Raspberry Pi
-      primary_links:
-        - links:
-            - title: Ubuntu for Raspberry Pi
-              description: Single board computer
-              url: /raspberry-pi
-              secondary_cta_title: Get Ubuntu for Raspberry Pi
-              secondary_cta_url: /download/raspberry-pi
-            - title: Ubuntu Core for Raspberry Pi
-              description: Embedded Linux
-              url: /download/raspberry-pi-core
-            - title: Ubuntu-ready Pi devices
-              description: Certified hardware
-              url: /certified?q=&limit=20&vendor=Raspberry+Pi+Foundation
```

Comment:
This should stay, with the first link just pointing to /download/raspberry-pi instead.